readme.md

```diff
@@ -117,6 +117,7 @@ Notable self-sovereign identity products and services that are in production.
   - [ethr DID Resolver](https://github.com/decentralized-identity/ethr-did-resolver) - Use Ethereum addresses as fully self managed Decentralized Identifiers and wrap them in a DID Document.
   - [web DID Resolver](https://github.com/decentralized-identity/web-did-resolver) - Use domains accessed through https as Decentralized Identifiers and retrieve an associated DID Document.
 - [Rust: did-key.rs](https://github.com/decentralized-identity/did-key.rs) - Provide basic support for `did:key` methods.
+- [Best-of-digital-identity](https://github.com/jruizaranguren/best-of-digital-identity) - Ranking, popularity and activity status of open-source digital identity projects, including decentralized identity projects.
```

Comment:
The "Tools and Technologies" section contains links to specific tools, libraries, and frameworks. Although very valuable, this addition is an overview of various digital identity-related projects and is not about one specific technology. I think the "Resources" section is more appropriate for a resource like this.

Reply:
I have changed it.